helm/reana/values.yaml

```diff
@@ -49,6 +49,20 @@ secrets:
   cern:
     sso: {}
   reana: {}
+  keycloak:
+    consumer_key: ""
+    consumer_secret: ""
+
+# Keycloak configuration (also refer to secrets.keycloak.*)
+keycloak_config:
+  enabled: false
+  title: ""
+  desciption: ""
+  base_url: ""
+  auth_url: ""
+  token_url: ""
+  userinfo_url: ""
+  realm: ""
```

Comment:
I see that we give administrators the possibility to provide the realm. Does this mean that you would like to support both providing `base_url` + `realm` or providing each URL separately?

Reply:
Probably not. Best to set everything manually, but for that, I still need to do some changes to the server.